lib/usb/usb_msc.c

```diff
@@ -468,6 +475,12 @@ static void scsi_inquiry(usbd_mass_storage *ms,
 			trans->csw.csw.dCSWDataResidue =
 				sizeof(_spc3_inquiry_response);
 
+			set_sbc_status_good(ms);
+		else if (1 == evpd) {
```

Comment:
You forgot a `}` here before the `else`

Reply:
which is what the travis build failure is from :) https://travis-ci.org/libopencm3/libopencm3/builds/382145893?utm_source=github_status&utm_medium=notification

Reply:
>You forgot a } here before the else

Thx! I fixed in fork.

Reply:
How do I update the pull request after fix? I already tried all the options =)

Reply:
```git push -f``` will auto-update the github PR.
(You may want/need to use ```git rebase -i``` to make the branch look how you want it first)